x/ccv/consumer/keeper/keeper.go

```diff
@@ -271,12 +283,25 @@ func (k Keeper) GetPacketMaturityTime(ctx sdk.Context, vscId uint64) uint64 {
 	return binary.BigEndian.Uint64(bz)
 }
 
-// DeletePacketMaturityTime deletes the the maturity time for a given received VSC packet id
+// DeletePacketMaturityTime deletes the packet maturity time for a given received VSC packet id
 func (k Keeper) DeletePacketMaturityTime(ctx sdk.Context, vscId uint64) {
 	store := ctx.KVStore(k.storeKey)
 	store.Delete(types.PacketMaturityTimeKey(vscId))
 }
 
+// ClearPacketMaturityTime deletes all the packet maturity time in the store
+func (k Keeper) ClearPacketMaturityTimes(ctx sdk.Context) {
```

Comment:
nit: "Clear..." is ambiguous since it suggests that the function is reseting the times. Why not use "Delete..." instead?